<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageSerialAnalysis</class>
 <widget class="QWidget" name="PageSerialAnalysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Plot Assistant</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0">
         <item>
          <widget class="QSplitter" name="splitter_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QSplitter" name="splitter">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>100</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QWidget" name="verticalLayoutWidget">
             <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
              <item>
               <widget class="QTextBrowser" name="cmdBrowser">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>1</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="verticalLayoutWidget_2">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QCustomPlot" name="generalPlot" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>50</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QWidget" name="verticalLayoutWidget_3">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QTabWidget" name="tabWidget_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="tabPosition">
                <enum>QTabWidget::East</enum>
               </property>
               <property name="tabShape">
                <enum>QTabWidget::Triangular</enum>
               </property>
               <property name="currentIndex">
                <number>0</number>
               </property>
               <widget class="QWidget" name="tab_5">
                <attribute name="title">
                 <string>Commands</string>
                </attribute>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_11">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                     <item>
                      <widget class="QLineEdit" name="sendCmdText_1">
                       <property name="readOnly">
                        <bool>false</bool>
                       </property>
                       <property name="placeholderText">
                        <string/>
                       </property>
                       <property name="clearButtonEnabled">
                        <bool>false</bool>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="sendCmdButton_1">
                       <property name="text">
                        <string>1</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_16">
                     <item>
                      <widget class="QLineEdit" name="sendCmdText_2"/>
                     </item>
                     <item>
                      <widget class="QPushButton" name="sendCmdButton_2">
                       <property name="text">
                        <string>2</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_15">
                     <item>
                      <widget class="QLineEdit" name="sendCmdText_3"/>
                     </item>
                     <item>
                      <widget class="QPushButton" name="sendCmdButton_3">
                       <property name="text">
                        <string>3</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_14">
                     <item>
                      <widget class="QLineEdit" name="sendCmdText_4"/>
                     </item>
                     <item>
                      <widget class="QPushButton" name="sendCmdButton_4">
                       <property name="text">
                        <string>4</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_13">
                     <item>
                      <widget class="QLineEdit" name="sendCmdText_5"/>
                     </item>
                     <item>
                      <widget class="QPushButton" name="sendCmdButton_5">
                       <property name="text">
                        <string>5</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_12">
                     <item>
                      <widget class="QLineEdit" name="sendCmdText_6"/>
                     </item>
                     <item>
                      <widget class="QPushButton" name="sendCmdButton_6">
                       <property name="text">
                        <string>6</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_11">
                     <item>
                      <widget class="QLineEdit" name="sendCmdText_7"/>
                     </item>
                     <item>
                      <widget class="QPushButton" name="sendCmdButton_7">
                       <property name="text">
                        <string>7</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_10">
                     <item>
                      <widget class="QLineEdit" name="sendCmdText_8"/>
                     </item>
                     <item>
                      <widget class="QPushButton" name="sendCmdButton_8">
                       <property name="text">
                        <string>8</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_9">
                     <item>
                      <widget class="QLineEdit" name="sendCmdText_9"/>
                     </item>
                     <item>
                      <widget class="QPushButton" name="sendCmdButton_9">
                       <property name="text">
                        <string>9</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_8">
                     <item>
                      <widget class="QLineEdit" name="sendCmdText_10"/>
                     </item>
                     <item>
                      <widget class="QPushButton" name="sendCmdButton_10">
                       <property name="text">
                        <string>10</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_7">
                     <item>
                      <widget class="QLineEdit" name="sendCmdText_11"/>
                     </item>
                     <item>
                      <widget class="QPushButton" name="sendCmdButton_11">
                       <property name="text">
                        <string>11</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_6">
                     <item>
                      <widget class="QLineEdit" name="sendCmdText_12"/>
                     </item>
                     <item>
                      <widget class="QPushButton" name="sendCmdButton_12">
                       <property name="text">
                        <string>12</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_5">
                     <item>
                      <widget class="QLineEdit" name="sendCmdText_13"/>
                     </item>
                     <item>
                      <widget class="QPushButton" name="sendCmdButton_13">
                       <property name="text">
                        <string>13</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <spacer name="verticalSpacer">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>40</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="tab_6">
                <attribute name="title">
                 <string>Channals</string>
                </attribute>
                <layout class="QVBoxLayout" name="verticalLayout_14">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_12">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                     <item>
                      <widget class="QCheckBox" name="checkBox">
                       <property name="layoutDirection">
                        <enum>Qt::LeftToRight</enum>
                       </property>
                       <property name="text">
                        <string>CH_0</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label">
                       <property name="frameShape">
                        <enum>QFrame::NoFrame</enum>
                       </property>
                       <property name="text">
                        <string>TextLabel</string>
                       </property>
                       <property name="textFormat">
                        <enum>Qt::AutoText</enum>
                       </property>
                       <property name="scaledContents">
                        <bool>false</bool>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_24">
                     <item>
                      <widget class="QCheckBox" name="checkBox_2">
                       <property name="text">
                        <string>CH_1</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_2">
                       <property name="text">
                        <string>TextLabel</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_23">
                     <item>
                      <widget class="QCheckBox" name="checkBox_3">
                       <property name="text">
                        <string>CH_2</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_3">
                       <property name="text">
                        <string>TextLabel</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_22">
                     <item>
                      <widget class="QCheckBox" name="checkBox_4">
                       <property name="text">
                        <string>CH_3</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_4">
                       <property name="text">
                        <string>TextLabel</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_21">
                     <item>
                      <widget class="QCheckBox" name="checkBox_5">
                       <property name="text">
                        <string>CH_4</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_5">
                       <property name="text">
                        <string>TextLabel</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_20">
                     <item>
                      <widget class="QCheckBox" name="checkBox_6">
                       <property name="text">
                        <string>CH_5</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_6">
                       <property name="text">
                        <string>TextLabel</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_19">
                     <item>
                      <widget class="QCheckBox" name="checkBox_7">
                       <property name="text">
                        <string>CH_6</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_7">
                       <property name="text">
                        <string>TextLabel</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_18">
                     <item>
                      <widget class="QCheckBox" name="checkBox_8">
                       <property name="text">
                        <string>CH_7</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_8">
                       <property name="text">
                        <string>TextLabel</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_17">
                     <item>
                      <widget class="QCheckBox" name="checkBox_9">
                       <property name="text">
                        <string>CH_8</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_9">
                       <property name="text">
                        <string>TextLabel</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_3">
                     <item>
                      <widget class="QCheckBox" name="checkBox_10">
                       <property name="text">
                        <string>CH_9</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_10">
                       <property name="text">
                        <string>TextLabel</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <spacer name="verticalSpacer_2">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>40</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="pageHelpButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../res.qrc">
               <normaloff>:/res/icons/Help-96.png</normaloff>:/res/icons/Help-96.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="cmdEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="cmdSendButton">
             <property name="text">
              <string>Send</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cmdClearButton">
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Unknow</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_13"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>widgets/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
